compute/cker/include/cker/operation/Reduce.h

```diff
@@ -330,7 +330,7 @@ class Reduce
       if (compute_sum)
       {
         // TODO(b/116341117): Eliminate float and do this completely in 8bit.
-        const float bias = -input_zero_point * scale * num_elements_in_axis + 0.5f;
```

Comment:
Just curious.
Why was there `+ 0.5f` in the old code? `bias` is used only for `std::round`, not `std::floor`. It's strange.

Reply:
I also don't know reason. TFLite 1.13 uses `+0.5f`, and it is removed now.